adapters/repos/db/inverted/tracker/propertyIdTracker_test.go

```diff
@@ -0,0 +1,190 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package tracker
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sync"
+	"testing"
+)
+
+func TestJsonPropertyIdTracker(t *testing.T) {
+	path := t.TempDir() + "/test.json"
+	defer os.Remove(path)
+	defer os.Remove(path + ".bak")
```

Comment:
if you're using `t.TempDir()` then it's not necessary to clean up the directories created, those tempory folders are being [clean up automatically](https://pkg.go.dev/testing#T.TempDir)

Please remove this lines